src/Arcus.Observability.Tests.Integration/Serilog/Sinks/ApplicationInsights/ExceptionTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.ApplicationInsights;
+using Microsoft.Azure.ApplicationInsights.Models;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using Xunit;
+using Xunit.Abstractions;
+using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+namespace Arcus.Observability.Tests.Integration.Serilog.Sinks.ApplicationInsights
+{
+    public class ExceptionTests : ApplicationInsightsSinkTests
+    {
+        public ExceptionTests(ITestOutputHelper outputWriter) : base(outputWriter)
+        {
+        }
+
+        [Fact]
+        public async Task LogException_SinksToApplicationInsights_ResultsInExceptionTelemetry()
+        {
+            // Arrange
+            string message = BogusGenerator.Lorem.Sentence();
+            var exception = new PlatformNotSupportedException(message);
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory())
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                // Act
+                logger.LogCritical(exception, exception.Message);
+            }
+
+            // Assert
+            using (ApplicationInsightsDataClient client = CreateApplicationInsightsClient())
+            {
+                await RetryAssertUntilTelemetryShouldBeAvailableAsync(async () =>
```

Comment:
We have exceptions here: `Operation returned an invalid status code 'InternalServerError'`

@stijnmoreels  What do we do with these tests then? Disable for now or?

Reply:
Hmm, maybe yes, if you create an issue, we can look for it on another time. The test seems correct so let's see.

Reply:
Here we go @stijnmoreels - Works now!